app/controllers/orders_controller.rb

```diff
@@ -1,82 +1,57 @@
 class OrdersController < ApplicationController
-  before_action :set_order, only: [:show, :edit, :update, :destroy]
 
-  # GET /orders
-  # GET /orders.json
+  before_action :get_order, except: [:index, :create, :new]
+  respond_to :html, :json
+
+  # skip_before_action :verify_authenticity_token
+
   def index
-    @orders = Order.all
-  end
+    @users = Order.all
+    respond_with(@orders) do |format|
+      format.json { render :json => @users.as_json }
```

Comment:
Please use the Ruby 1.9 hash literal syntax when your hash keys are symbols. 
https://github.com/bbatsov/ruby-style-guide#hash-literals